### ISSUE TITLE
New Parameter to choose Slot via EasyFlash 3 USB Utilities

### ISSUE THREAD

**Issue by soenkeraufeisen** [KimJorgensen/easyflash#1]
It would be great to select the Slot via EasyFlash 3 USB Utilities at EasyProg.
Example at PC:

easyflash -b turrican.crt -s 5

So, the crt will be copied to Slot 5.

**Comment by KimJorgensen** [comment]
Yeah, that would be great. I guess there could also be an option for selecting a kernal or freezer slot.
Did you look into how the USB protocol can be extended with additional parameters?
